content/news/Reproc-Update-1-C2-C13-C3.md

```diff
@@ -0,0 +1,79 @@
+Title: K2 Campaigns 2, 3, and 13 Reprocessed
```

Comment:
`Reprocessed` => `reprocessed`  (we have a convention to use lower case in blog titles)